charts/kube-vip-cloud-provider/templates/deployment.yaml

```diff
@@ -12,13 +12,20 @@ spec:
     metadata:
       labels:
         {{- include "kube-vip-cloud-provider.selectorLabels" . | nindent 8 }}
+      annotations:
+        configmap-checksum: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
```

Comment:
Is this related to https://helm.sh/docs/howto/charts_tips_and_tricks/#automatically-roll-deployments  ?

I did a test and confirmed that kube-vip-cloud-provider will read updates to the configmap if it changes, so it doesn't need to be restarted if the configmap changes. This is consistent with the log messages indicating it uses `watch` to detect configmap changes.

Reply:
Thanks for the confirmation. I removed the configmap-checksum annotations